app/services/metrc_service/harvest.rb

```diff
@@ -48,7 +48,7 @@ def build_harvest_plants_payload(items, batch)
           ActualDate: harvest_date,
           Plant: item.relationships.dig('barcode', 'data', 'id'),
           Weight: average_weight,
-          UnitOfWeight: item.attributes['harvest_unit'],
```

Comment:
@sd @betacar I'm trying to replace this with a reference to the `resource_unit` but there is no `resource_unit_id` for Item [(see items API response here)](https://github.com/artemis-ag/apps-cannapis/pull/22/files#diff-cf26d231da2367545c2f018b58a4f37cR1). Should this instead reference one of the `resources` in the harvest completion? If so, how do I resolve which one (in this example there are 4 `resources` with differing `resource_unit_id`s in the completion payload [(here)](https://github.com/artemis-ag/apps-cannapis/pull/22/files#diff-6344f85ce32f77eb9a55914162d989beR233)

Reply:
@dandehavilland For now, you'll have to request the facility's resource and look the one whose name has `wetweight` (@sd please correct me if my name nomenclature is wrong). That's the one you should use.

This is a temporary solution, while the API is improved for these cases.

Reply:
OK, thanks @betacar. I have units by the names of `g of Wet Material - Boss Hog` and `g of Wet Material - 5th Element` so I'll filter based on `/Wet Material/`.

Reply:
@betacar Having said that, I don't see either of those in a `harvest` completion or any of the associated `generate` completions.

Reply:
Look for them in `process` completions out of `move` actions.

In general, we should look for both `process` and `generate` completions as triggers. Using one or the other will depend on the exact workflow for a customer (using separate stages between some of these phases or not)